### PR TITLE
Change of spons/sponors folder name to spons/frontpage-sponsors

### PR DESCRIPTION
**Pull request by punit121**
names changed

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -4,16 +4,78 @@
...
+      <img class="img-responsive"src="img/spons/frontpage-sponsors/g8.png"width="148" height="148">
     </div>
+  <div class="item">
+      <img src="img/spons/frontpage-sponsors/strategic.png"width="148" height="180">
+    </div>
+  <div class="item">
+      <img src="img/spons/frontpage-sponsors/g9.png"width="148" height="148">
+    </div>
+  <div class="item">
+      <img class="img-responsive"src="img/spons/frontpage-sponsors/e7.png"width="200" height="400">
```
Comment (icyflame) [inline index.php]: @punit121

Add a space before the SRC attribute. and also before the width attribute.

**Comment by punit121** [comment]
spaces between src and width done

**Comment by icyflame** [comment]
Do remember to use an HTML code beautifying editor plugin, next time onwards.

**Comment by icyflame** [comment]
@punit121 it seems that you have not pulled from my this repo, before making those changes. Always, pull and update your local repo, before starting to make changes. Please pull now, and update this PR.

**Pull request closed without merging**